src/kakarot/precompiles/precompiles.cairo

```diff
@@ -66,6 +66,7 @@ namespace Precompiles {
             sub_context=cast(0, model.ExecutionContext*),
             destroy_contracts_len=0,
             destroy_contracts=cast(0, felt*),
+            read_only=calling_context.read_only,
```

Comment:
This does not really make sense as this execution context is stopped at construction and is actually only a “placeholder” for the precompile returned data. I’d put false/0 instead

Reply:
Great point at being stopped at construction. On it!